apollo-client/src/main/java/com/ctrip/framework/apollo/spring/boot/ApolloApplicationContextInitializer.java

```diff
@@ -131,6 +137,27 @@ private void fillSystemPropertyFromEnvironment(ConfigurableEnvironment environme
 
     System.setProperty(propertyName, propertyValue);
   }
+  
+  private void fillSystemPropertyFromEnvironment(ConfigurableEnvironment environment, String propertyName, String name) {
```

Comment:
I would suggest we leave the 'app.id' logic as what it was and add extra logic for `apollo.app.id`. And I'm still not convinced whether it's necessary to support `apollo.app.id`.